tests/PHPStan/Rules/Functions/data/bug-9697.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Bug9697;
+
+
+function doFoo(): void
+{
+	$oldItems = [1,2,3];
+	$newItems = [1,2];
+
+	$comparator = fn (int $a, int $b):int => $a - $b;
```

Comment:
interessting.. I would expect the php 7.2/7.3 lint jobs to error on this file

https://3v4l.org/FrKmK